bundles/binding/org.openhab.binding.mpd/src/main/java/org/openhab/binding/mpd/internal/MpdBinding.java

```diff
@@ -219,10 +219,20 @@ private void executePlayerCommand(String playerCommandLine, Object commandParams
                         player.stop();
                         break;
                     case VOLUME_INCREASE:
-                        player.setVolume(player.getVolume() + VOLUME_CHANGE_SIZE);
+                        int result = player.getVolume() + VOLUME_CHANGE_SIZE;
+                        if (volumeInvalid(result)) {
+                            logger.warn("Command ignored because the resulting volume would be invalid: {}", result);
+                        } else {
+                            player.setVolume(result);
+                        }
                         break;
                     case VOLUME_DECREASE:
-                        player.setVolume(player.getVolume() - VOLUME_CHANGE_SIZE);
+                        result = player.getVolume() - VOLUME_CHANGE_SIZE;
+                        if (volumeInvalid(result)) {
+                            logger.warn("Command ignored because the resulting volume would be invalid: {}", result);
+                        } else {
+                            player.setVolume(result);
+                        }
```

Comment:
Indeed, you're correct. I should've checked the player class to see what it was doing with the values.

I'll undo this change.